collector/filesystem_common.go

```diff
@@ -70,7 +71,9 @@ func init() {
 // NewFilesystemCollector returns a new Collector exposing filesystems stats.
 func NewFilesystemCollector(logger log.Logger) (Collector, error) {
 	subsystem := "filesystem"
+	level.Info(logger).Log("msg", "Parsed flag --collector.filesystem.ignored-mount-points", "flag", *ignoredMountPoints)
```

Comment:
Why not debug?

Reply:
I'm trying to surface this information to make it more obvious for https://github.com/prometheus/node_exporter/issues/1737.

Reply:
thanks